src/components/FileMenu/SaveAsDialog.js

```diff
@@ -16,7 +16,13 @@ import { supportedFileTypes, labelForFileType } from './utils.js'
 const NAME_MAXLENGTH = 230
 
 export const SaveAsDialog = ({ type, object, onClose, onSaveAs }) => {
-    const [name, setName] = useState(object?.name)
+    const [name, setName] = useState(
+        object?.name
```

Comment:
We should probably use `displayName` if available.
It should always be, but we must take into account the case of apps renaming `displayName` to `name` when fetching the visualization.

Reply:
`displayName` with fallback to `name` 👍